packages/block-library/src/rss/block.json

```diff
@@ -46,6 +46,26 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"color": {
+			"background": true,
+			"text": true,
+			"gradients": true,
+			"link": true
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontFamily": true,
+			"__experimentalFontWeight": true,
+			"__experimentalFontStyle": true,
+			"__experimentalTextTransform": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalTextDecoration": true,
```

Comment:
Probably not a big deal, but the underline control in text decoration won't affect `.wp-block-rss__item-title > a` elements as they're being directly controlled by this rule:

```css
a:where(:not(.wp-element-button)) {
    text-decoration: underline;
}
```
I'm not sure whether users will expect text decoration to affect all text within the RSS block 🤔

Reply:
There might be some options here to apply text decoration more effectively. 

Individual block support features, e.g. text-decoration, can have their serialization of styles and classes skipped. They can then be manually applied to the appropriate inner elements. For the Global Styles side of that equation, the [Block Selectors API](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-selectors/) allows custom selectors for individual features so they target the same inner elements.

For an example of skipping serialization of only`textDecoration`, see the [Navigation block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation/block.json?plain=1#L119). 

_Note: The navigation block skipped that serialization before the Selector's API was implemented so I'm not sure if it covers Global Styles or not._

Reply:
Thanks @aaronrobertshaw for the reference and suggestion.

Tried to follow the same way as in the Navigation block. Bu as you mentioned, it doesn't covers Global Styles. We need to tweak more on this.

I have added all the changes in recent commit.

More suggestions are welcome for this scenario.